gensrc/thrift/Descriptors.thrift

```diff
@@ -100,6 +100,8 @@ struct TTextFileDesc {
     
     // escape character
     8: optional i8 escape
+
+    9: optional i32 skip_header_line_count
```

Comment:
After adding this Skip, will it cause problems with the optimizer, such as Limit 1. In the past, FE would select 1 file, but with this skip, problems may occur?